tests/integration/account_balance_query_e2e_test.py

```
@@ -1,14 +1,29 @@
     try:
         CryptoGetAccountBalanceQuery(account_id=env.operator_id).execute(env.client)
     finally:
-        env.close()
\ No newline at end of file
+        env.close()
+
+@pytest.mark.integration
+def test_integration_contract_balance_query_can_execute():
+    env = IntegrationTestEnv()
+
+    contract_id_str = os.getenv("HEDERA_TEST_CONTRACT_ID")
```

Ruff (0.14.10)

Undefined name `os`

(F821)